onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -553,7 +609,7 @@ def parse_arguments():
                         nargs="+",
                         type=str,
                         default=['onnxruntime'],
-                        choices=['onnxruntime', 'torch', 'torchscript'],
+                        choices=['onnxruntime', 'onnxruntime_with_io_binding', 'torch', 'torchscript'],
```

Comment:
How about adding an option --disable_io_binding instead of adding as a new engine?
Then we can add a column named IO_Binding (similar to float16 or optimizer) in result file.

Reply:
Added. Also changed the option in benchmark_gpt2.py for consistency.